packages/core/src/types.ts

```diff
@@ -1325,3 +1328,23 @@ export type InterpreterFrom<
 >
   ? Interpreter<TContext, TStateSchema, TEvent, TTypestate>
   : never;
+
+export interface ActorContext<TEvent extends EventObject, TEmitted> {
```

Comment:
mentioning "actor context" will become ambiguous now 😬 before this PR I would totally think of a particular machine-actor's context value.

Reply:
I can't think of a better name - `context` is what's used by Akka

Reply:
@mattpocock @farskid maybe you would have some names for this?

Reply:
Reference: https://doc.akka.io/japi/akka/current/akka/actor/ActorContext.html

My opinion: I'm fine with calling it `ActorContext` in the code but referring to it in the documentation just as an object, e.g.:

> You can access the parent from the 3rd argument:
> ```js
> const transition = (state, event, { parent }) => {
>   // ...
> };
> ```